gui/wxpython/web_services/widgets.py

```diff
@@ -230,6 +230,12 @@ def _requestPage(self):
 
         self.req_page_panel.SetSizer(self.req_page_sizer)
 
+        size = list(self.req_page_sizer.GetMinSize())
```

Comment:
This seems little weird, I would rather set the minimum height as fixed e.g.:

    self.SetMinSize((-1, 300))

Reply:
I'm using Gentoo distro (`4.0.7.post2 gtk3 (phoenix) wxWidgets 3.0.4`) with [dynamic window manager](https://dwm.suckless.org/) (main difference against Ubuntu distro) for X. It's work properly on Ubuntu 18.04 distro (dialog height expand to max desktop visible height).